### PR TITLE
Quiz scaffolding

### PR DESCRIPTION
**Pull request by btburton42**

### CODE REVIEW

File: app/components/Question/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+/* component styles */
+import { styles } from './styles.scss';
+
+const Question = ({question}) => {
```
Comment (normangilmore) [inline app/components/Question/index.js:7]: I find this new'ish React way to create stateless components by using plain ES6 arrow functions to be quite obscure. I feel like they should be annotated with a comment `/* Create stateless react component */` so it's easier to realize instantly you are looking at a React render method.

File: app/containers/Quiz/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import {styles} from './styles.scss';
+
+const mapStateToProps = state => {
+  return {
+    question: state.quiz.question
+  };
+}
+
+@connect (
```
Comment (normangilmore) [inline app/containers/Quiz/index.js:19]: The `@connect` syntax is being enabled by `babel-plugin-transform-decorators-legacy`, but technically decorators will be part of ES7, so is not standard yet, right? Wonder if we should stick with ES6 syntax until we officially adopt ES7 when it's ready. Just for simplicity so people don't need to look up what exactly this is. Just a thought.

**Review by normangilmore — APPROVED** [review]
Looks good, thank you for prepping this.

**Comment by normangilmore** [comment]
:+1: for good measure.

File: app/components/Question/index.js (hunk 1 above)
Comment (btburton42) [reply app/components/Question/index.js:7]: fair points. i will add a comment. however, this is the way i am creating components these days as it removes any questions of state, props, etc and by definition makes this a slave component to the parent container.

File: app/containers/Quiz/index.js (hunk 2 above)
Comment (btburton42) [reply app/containers/Quiz/index.js:19]: they are certainly not standard (yet) but they are too powerful and fun to play around with to go back to the dirty way IMO. i would argue to leave them in for now and if they prove troublesome or confusing we can revert.

**Comment by btburton42** [comment]
thanks for the feedback @normangilmore i appreciate it. going to go ahead and merge so the rest of the team is unblocked.

**Pull request merged**